web/src/components/Dashboard/RequestPage/Term.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { Component } from 'react';
+
+ 
+class Term extends Component {
```

Comment:
What is the purpose of this component? I didn't see it being used anywhere.
btw, it's better to write "const Term = () {}" instead of "class Term extends ..."

Reply:
Deleted Class "Term" since it's not needed at the moment